releng/prepare_release_branch.py

```diff
@@ -54,7 +54,7 @@ def delete_dead_branch(branch_path, current_version):
 
 
 def rotate_files(rotator_bin, branch_path, current_version):
-    suffixes = ['beta', 'stable1', 'stable2', 'stable3']
+    suffixes = ['stable1', 'stable2', 'stable3', 'stable4']
```

Comment:
making this change without making a corresponding change to all of the non-generated jobs with the word 'beta` in them is confusing